app/lib/frontend/templates/views/shared/detail/header.dart

```diff
@@ -44,7 +48,25 @@ d.Node detailHeaderNode({
                   ),
                 ),
                 d.img(
-                  classes: ['ff-banner', 'ff-banner-mobile'],
+                  classes: [
```

Comment:
Should we consider using a `<picture>` instead of classes to avoid loading both images?
Like suggested in https://stackoverflow.com/a/74639288

Reply:
`<picture>` cannot select (`<source>`) the image based on the presence of the top-level CSS class. We could use it for desktop/mobile view switch, but then we would need to emit the hardcoded width rules into the HTML `<source media="">` selector, which would complicate it a bit (as we have that by using SCSS variables, and we don't have a direct access to it while rendering the HTML).

Reply:
In practice `<picture>` is not better for such use case, it only has a different tradeoff (which we could choose, but we may as well stick with the current practice).